pkg/client/unversioned/request.go

```diff
@@ -48,6 +48,16 @@ import (
 // are therefore not allowed to set manually.
 var specialParams = sets.NewString("timeout")
 
+// backoff is the underlying backoff struct we use to prevent thundering herd drama, esp when this runs in a controller.
+var backoff = &URLBackoff{Backoff: util.NewBackOff(1*time.Second, 120*time.Second)}
```

Comment:
Ok, for now I can implement this if we want.   right now its in the client, but i can move it into the VERB().    I will redo graphs afterward this to see how much  of an improvement, and we can always move back into client after experimenting on per-request backoffs.